Gopkg.toml

```diff
@@ -45,3 +45,7 @@ required = [
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "1.3.0"
+
+[[override]]
+  name = "github.com/russross/blackfriday"
```

Comment:
What is this dependency used for? I do not see it imported anywhere.

Reply:
This is used implicitly by `github.com/spf13/cobra` -> `github.com/cpuguy83/go-md2man` -> `github.com/russross/blackfriday`, and it's not working with the latest version.

Reply:
I see, strange that they do not lock in the version that works on cobra...